benchmark/sirun/exporting-pipeline/meta.json

```diff
@@ -0,0 +1,10 @@
+{
+  "setup": "nohup node ../../e2e/fake-agent.js",
+  "run": "node index.js",
+  "cachegrind": true,
+  "iterations": 10,
```

Comment:
Would less iterations avoid random failures due to timeout? Otherwise removing cachegrind and temporarily losing instruction count.